src/Augmentation/data/AugmentationCreator.tsx

```diff
@@ -117,10 +117,10 @@ export const initSoAAugmentations = (): Augmentation[] => [
     repCost: 1e4,
     moneyCost: 1e6,
     info:
-      "Extra-occular neurons taken from old martial art master. Injecting the user the ability to " +
-      "predict enemy attack before they even know it themself.",
+      "Extra-occular neurons taken from old martial art master. Injecting them gives the user the ability to " +
+      "predict the enemy's attack before they even know it themself.",
```

Comment:
makes the sentence clearer as to what is 'injected' and 'enemy attack' doesn't make sense in this context